tokenMetadata/0.json

```diff
@@ -0,0 +1,7 @@
+{
+  "name": "Simon",
+  "image": "https://ipfs.io/ipfs/QmZt5S8tD7L4nMBo4NTtVDpV3qpteA1DXJwKRmuF318tHd",
+  "type": "Gorilla",
+  "artist": "Matty Fraser",
+  "description": "Original Gorilla\n\nSimon, named after Simon de la Rouviere originated from the deep dark github jungle, and his code 'this artwork is always on sale' inspired this project. He is a wizard who put us on the right track. He has helped inform some of our design decisions regarding our subsequent smart contracts, and always been willing to lend some great advice or checkout a GitHub issue for us.\n\nFunds raised by Simon, one of the first wildcards, flow toward the Wild Tomorrow Fund, our first conservation partner."
```

Comment:
Maybe the funds from Simon won't always flow to Wild Tomorrow Fund but eventually another or group of organisations? Maybe a thought to change

Reply:
Yeah, good point. We can write that on the website, but it shouldn't be in the metadata I don't think

Reply:
I'll change it.

Before: "Funds raised by Simon, one of the first wildcards, flow toward the Wild Tomorrow Fund, our first conservation partner."
After: "Funds raised by Simon initially go to Wild Tomorrow Fund, but Simon is really a 'wildcards' gorilla."

We could also say: "In the future funds generated by Simon may be managed by a DAO."